.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,15 +8,13 @@ Thanks for your interest in plotly.js!
 - You should only update files in `lib/locales/`, not those in `dist/`
 
 ### Features, Bug fixes, and others:
-
-Developers are strongly encouraged to first make a PR to their own plotly.js fork and ask one of the maintainers to review the modifications there. Once the pull request is deemed satisfactory, the developer will be asked to make a pull request to the main plotly.js repo and may be asked to squash some commits before doing so.
-
 Before opening a pull request, developer should: 
 
 - `git rebase` their local branch off the latest `master`,
 - make sure to **not** `git add` the `dist/` folder (the `dist/` is updated only on version bumps),
-- make sure to commit changes to the `package-lock.json` file (if any),
+- make sure to commit changes to the `package-lock.json` file (if any new dependency required),
 - write an overview of what the PR attempts to do,
+- make a PR directly from their fork to the main repository i.e. plotly/plotly.js
```

Comment:
Let's just remove all mention of which copy of the repo to make a PR to - I think at this point it's standard practice to PR to the main one. So that would be removing this line, as well as line 5 re: translations.